cmd/getbackup/getbackup.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-02-01/storage"
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/log"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	azureclientstorage "github.com/openshift/openshift-azure/pkg/util/azureclient/storage"
+	"github.com/openshift/openshift-azure/pkg/util/nodeconf"
+)
+
+var (
+	logLevel    = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
```

Comment:
Keep the help messages across all flags consistent: start with a capital letter.

Reply:
thanks